dis_snek/ext/paginators.py

```diff
@@ -286,6 +287,26 @@ async def send(self, ctx: Context) -> Message:
 
         return self._message
 
+    async def reply(self, ctx: MessageContext) -> Message:
+        """
+        Reply this paginator to ctx.
+
+        Args:
+            ctx: The context to reply this paginator with
+
+        Returns:
+            The resulting message
+
+        """
+        self._message = await ctx.reply(**self.to_dict())
+        self._author_id = ctx.author.id
+
+        if self.timeout_interval > 1:
+            self._timeout_task = Timeout(self)
+            self.client.loop.create_task(self._timeout_task())
```

Comment:
> It looks like you created this using an older version of dis_snek. `client.loop` no longer exits. I suggest updating your branch, and using `asyncio.create_task` instead.

does it work if I just edit the line?

I accidentally based this PR on `main` and not on `dev`.
Otherwise I would reopen this PR but with the code based on `dev`.

Reply:
The latter would be best, tbh

Reply:
Ok, I'll close the PR and reopen it in the evening when I'm home (UTC+1 for context)